### PR TITLE
Chore: update_humans_txt

### PR DESCRIPTION
**Pull request by andykwoods**
## Description

Updating Humans.txt for all the recent new joiners

### CODE REVIEW

File: public/humans.txt [hunk 1]
```diff
@@ -122,8 +143,7 @@ Oak National Academy’s tech platform has been created by a team of humans supp
   Site: https://capeflair.com
   From: Guildford, UK
 
-  Verity Roberts (Product Manager)
-  From: Cambridge, UK
+
```
Comment (m-jenic) [inline public/humans.txt:146]: Introduced whitespace

**Review by abisolakuteyi — COMMENTED** [review]
typo on line 28

File: public/humans.txt [hunk 2]
```diff
@@ -8,17 +8,25 @@ Oak National Academy’s tech platform has been created by a team of humans supp
...
   LinkedIn: https://www.linkedin.com/in/andykwoods/
   From: Dublin, Ireland
 
   Ben Protheroe (Software Engineer)
   From: London, lives in Cornwall
 
+  Carl Whittaker (Software Engineer)
+  LinkedIn: https://www.linkedin.com/in/carlmw 
+  From: London, UK
+
```
Comment (carlmw) [inline public/humans.txt:29]: @andykwoods I think I definitely belong in the thanks section as a contractor, otherwise I should probably be removed.

**Review by carlmw — APPROVED** [review]

**Pull request merged**